glimpseClientVersion.js

```diff
@@ -0,0 +1,6 @@
+'use strict'
```

Comment:
The problem is that there's no guarantee that this file will get re-committed when regenerated.  For example, if I cut a release, I first update the version number in `package.json`.  I don't do a build myself, because when that PR is merged I'll then create a release in GitHub.  The CI system will do the actual build that generates the NPM package (indirectly updating that file), but the CI system won't submit that change back to the repo.